tests/descriptors/topological.py

```diff
@@ -0,0 +1,164 @@
+import pytest
+from rdkit.Chem import MolFromSmiles, Mol, GetDistanceMatrix
+
+from skfp.descriptors import topological as top
+
+
+def diverse_molecules() -> list[tuple[str, Mol]]:
+    return [
+        ("ethane", MolFromSmiles("CC")),
+        ("ethanol", MolFromSmiles("CCO")),
+        ("carbon_dioxide", MolFromSmiles("O=C=O")),
+        ("benzene", MolFromSmiles("C1=CC=CC=C1")),
+        ("acetic_acid", MolFromSmiles("CC(=O)O")),
+        ("pyridine", MolFromSmiles("c1ccncc1")),
+        ("isobutane", MolFromSmiles("C(C)(C)C")),
+        ("pyrimidine", MolFromSmiles("c1cnc2ncnc12")),
+    ]
+
+
+@pytest.mark.parametrize(
+    "mol_name, expected_value",
+    [
+        ("ethane", 1),
+        ("ethanol", 4),
+        ("carbon_dioxide", 4),
+        ("benzene", 27),
+        ("acetic_acid", 9),
+        ("pyridine", 27),
+        ("isobutane", 9),
+        ("pyrimidine", 55),
+    ],
+)
+def test_weiner_index(mol_name, expected_value):
+    mol_tuple = next(item for item in diverse_molecules() if item[0] == mol_name)
+    mol = mol_tuple[1]
```

Comment:
Just change `diverse_molecules` to return `dict[str, Mol]` and then you can simply select by name

Reply:
In fact, this should simply all tests here